cmd/relayer_exporter/relayer_exporter.go

```diff
@@ -45,20 +47,25 @@ func main() {
 
 	log.Info(
 		fmt.Sprintf(
-			"Getting IBC paths from %s/%s/%s on GitHub",
+			"Github IBC registry: %s/%s",
 			cfg.GitHub.Org,
 			cfg.GitHub.Repo,
-			cfg.GitHub.IBCDir,
 		),
+		zap.String("Mainnet Directory", cfg.GitHub.IBCDir),
+		zap.String("Testnet Directory", cfg.GitHub.TestnetsIBCDir),
 	)
 
+	ctx := context.Background()
 	// TODO: Add a feature to refresh paths at configured interval
-	paths, err := cfg.IBCPaths()
+	paths, err := cfg.IBCPaths(ctx)
 	if err != nil {
 		log.Fatal(err.Error())
 	}
 
-	rpcs := cfg.GetRPCsMap()
+	rpcs, err := cfg.GetRPCsMap(paths)
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
Validating a config is a nice feature but what if someone wants to use the exporter only for some of the paths? Maybe we should introduce a flag to validate and terminate exporter by default but give also an option to log a warning and continue if some RPCs are missing in the config. It will make an exporter useful for someone using it only for `cosmos_wallet_balance` metric

Reply:
Might be a good idea to have a flag but perhaps warrants a separate PR when the need arrives? I would like to keep the scope of this PR limited to current usecase